test/unity/toolkits/drawing_classifier/test_dc_serialization.cxx

```diff
@@ -162,7 +162,8 @@ BOOST_AUTO_TEST_CASE(test_export_coreml) {
        {"warm_start", ""},
        {"feature", features[0]}});
 
-  auto ml_model_wrapper = dc.export_to_coreml("", /* debug no throw */ true);
```

Comment:
Gah! Sorry. I should have checked all the C++ tests before building.

Reply:
build `all` could tell you during the compile stage.

Reply:
Good point. I wonder if we should just build all the C++ unit tests as part of the travis build.